examples/custom-animation/running-dashed-message-flow/README.md

```diff
@@ -0,0 +1,39 @@
+# Running Dashed Message Flow
+
+Javascript example to demonstrate how to add custom running dashed animation on a Message Flow of the BPMN Diagram.
+- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/custom-animation/running-dashed-message-flow/index.html)
+- to run locally, open the [index.html](index.html) directly in a Web Browser
+
+## ♻️ Usage
+
+After the main `BpmnVisualization` instantiation, get the HTML element corresponding to the message flow to add custom CSS class for the animation.
+````css
+    .running-dashed > path:nth-child(2) {
+        /* The corresponding SVG element for the line of the edge:
+            <path d="M 719.24 319.13 L 786.53 319.13" fill="none" stroke="black" stroke-width="1.78" stroke-miterlimit="10" pointer-events="stroke"></path>
```

Comment:
❓ why not only explaining we target the 1st path node of the svg group which draws the line? this will be simplier

Reply:
Because I'm not sure at 100% that the corrsponding path is at the second place every time.